pkg/controller/controller.go

```diff
@@ -101,7 +101,6 @@ var (
 //TODO consolidate ane librarize
 func logGRPC(ctx context.Context, method string, req, reply interface{}, cc *grpc.ClientConn, invoker grpc.UnaryInvoker, opts ...grpc.CallOption) error {
 	glog.V(5).Infof("GRPC call: %s", method)
-	glog.V(5).Infof("GRPC request: %+v", req)
```

Comment:
I was wondering about the exact same issue today, independently of you.

Removing the entire line is too drastic. It also removes less sensitive information.

My conclusion was that logging `req` entirely is probably okay and may be desired when someone explicitly chooses a high log level. However, such a high log level should not be used in production, which implies adding warnings about it somewhere.

Alternatively, formatting of req could be changed so that it masks the actual secret values.

Note that a similar logGRPC implementation also exists elsewhere (other sidecar apps, the hostpath driver itself). Changing just this one line here would not be enough. I am working on refactoring the code so that logGRPC lives in csi-common.

Reply:
That seems like the way to go to me too. The TODO suggests the same thing. Let me know if I can help in any way with the refactor 😄

Reply:
Let's discuss the code refactoring in https://github.com/kubernetes-csi/drivers/issues/81